### DISCUSSION TITLE
Why cluster mode is much slower than standalone mode

### DISCUSSION THREAD

**Discussion by 2435043xia** [redis/go-redis#1898] [Q&A]
even ping is much slower
![image](https://user-images.githubusercontent.com/14746292/133933061-e26ea7fe-0ef4-48fe-bb7e-407fbf56fa8a.png)
![image](https://user-images.githubusercontent.com/14746292/133933108-32e029c7-d554-445f-b9c9-3bce4103e65a.png)
![image](https://user-images.githubusercontent.com/14746292/133933089-aff47813-b2c8-4ac7-9889-ebd97ac0b058.png)
Above is cluster, below is standalone

Can anyone tell me why?

**Comment by wziww** [comment]
Cant reproduce this in `v8@v8.11.3`.
And the `ping` command will be sent to cluster nodes in random, which means your test result may be influenced?

**Reply by 2435043xia** [reply]
> Cant reproduce this in `v8@v8.11.3`.
> And the `ping` command will be sent to cluster nodes in random, which means your test result may be influenced?

This is so cool,Thank you for solving my problems that bothered me for a few days.

My version is v6, everything is normal after changing to v8
